README.md

```diff
@@ -352,18 +354,36 @@ buildctl build ... --import-cache type=local,src=path/to/input-dir
 
 The directory layout conforms to OCI Image Spec v1.0.
 
+#### GitHub Actions cache
```

Comment:
Should mark that this is currently experimental. Maybe link for the issue with TODO items.